src/pages/App.tsx

```diff
@@ -3,7 +3,7 @@ import { Route, Switch } from 'react-router-dom'
 import styled from 'styled-components'
 import GoogleAnalyticsReporter from '../components/analytics/GoogleAnalyticsReporter'
 import AddressClaimModal from '../components/claim/AddressClaimModal'
-import Header from '../components/Header'
+import Header from 'components/Header'
```

Comment:
unfortunate side-effect, have to change to abs path :disappointed:

Reply:
Yes, I know. This could bring us some conflicts. But is fairly easy to solve if they ever happen